controllers/admin/AdminSelfUpgradeController.php

```diff
@@ -236,7 +234,7 @@ public function init()
                 // delete the potential xml files we saved in config/xml (from last release and from current)
                 $upgrader->clearXmlMd5File($this->upgradeContainer->getProperty(UpgradeContainer::PS_VERSION));
                 $upgrader->clearXmlMd5File($upgrader->getDestinationVersion());
-                Tools14::redirectAdmin(self::$currentIndex . '&conf=5&token=' . Tools14::getValue('token'));
+                Tools14::redirectAdmin($this->context->link->getAdminLink('AdminSelfUpgrade', false) . '&conf=5&token=' . Tools14::getValue('token'));
```

Comment:
Actually this whole block can be deleted, the action `refreshCurrentVersion` is not anymore sent.

It has disappeared on v7 with the rework of the UI, can you please delete the lines from l. 223 to 238?